### DISCUSSION TITLE
Plugin doesn't work in IDEA

### DISCUSSION THREAD

**Discussion by troyaqgis** [kdbinsidebrains/plugin#55] [Q&A] [answered]
I have a problem with plugin in IntelliJ IDEA (I checked v. 2022.2.3. and 2020.3.1)
What is the reason? 

![obraz](https://user-images.githubusercontent.com/36294360/197032376-0a44692a-8902-46a3-9941-79f96e538e20.png)
![obraz](https://user-images.githubusercontent.com/36294360/197032575-60d91224-f1b4-4eda-870e-23e812637efb.png)
![obraz](https://user-images.githubusercontent.com/36294360/197032647-c7d3a77b-a7df-42fb-a418-64b850498b6a.png)
![obraz](https://user-images.githubusercontent.com/36294360/197032757-a7750b48-5e40-4fa6-8b71-9fb2e05579df.png)
![obraz](https://user-images.githubusercontent.com/36294360/197032801-9b537a0f-49bc-4e5d-bd48-72f95ed2f64a.png)
![obraz](https://user-images.githubusercontent.com/36294360/197032847-ed175024-a2fa-420d-8015-4397a39503d5.png)

![obraz](https://user-images.githubusercontent.com/36294360/197032968-c8568604-9b67-4b45-91c2-b41f6f2d861c.png)

**Comment by smklimenko** [comment, marked as answer]
You have another plugin associated with 'q' language: q by Antonio Andrade. They can not work together at the same time as both pointed to the same language.

You have to disable KdbInsideBrains, Disable/Remove 'q' plugin and enable 'KdbInsideBrains' again.

**Reply by troyaqgis** [reply]
Thanks, it works:)
